cabal-testsuite/PackageTests/ConditionalAndImport/cabal.project

```diff
@@ -0,0 +1,3 @@
+packages: .
+
+import extra.project
```

Comment:
`.cabal` files use fields, not section headers.

My reasoning was that why to allow having

```
import foo
  something-in-section: ...
```

to begin with.

Reply:
you're commenting on an older version. with the parser cleanup this uses `import: foo` as you seem to desire.

Reply:
:+1: